MiniApp/Classes/Extensions/URLSession+SessionProtocol.swift

```diff
@@ -0,0 +1,14 @@
+extension URLSession: SessionProtocol {
+    func startDataTask(with request: URLRequest, completionHandler: @escaping (Result<ResponseData, Error>) -> Void) {
+        dataTask(with: request) { (data, response, error) in
+            if let error = error {
+                return completionHandler(.failure(error))
+            }
+            guard let httpResponse = response as? HTTPURLResponse, let data = data else {
+                let error = NSError(domain: "APIClient", code: (response as? HTTPURLResponse)?.statusCode ?? 0, userInfo: [NSLocalizedDescriptionKey: "Unknown server error occurred"])
```

Comment:
For Kotlin, we split up long lines similar to this:
```
let error = NSError(
    domain: "APIClient", 
    code: (response as? HTTPURLResponse)?.statusCode ?? 0, 
    userInfo: [NSLocalizedDescriptionKey: "Unknown server error occurred"]
) 
```
Is there a similar convention in Swift? I could only find a Google style guide for Swift: https://google.github.io/swift/#line-wrapping

Does Apple have a style guide?

Reply:
I was looking at the Xcode text editing preferences, looks like there is no option available for line-wrapping functions. I will follow the guideline from google which looks good.

Reply:
As long as you're consistent I don't think it matters.
I think it's more usual to line break after the first param and align the start of the params below that.